tests/test_helpers.py

```diff
@@ -474,62 +474,44 @@
       "--date", "2020-04-29", "--collaborators", "afriend", "--description", "more work",
       "--group-members", "ascopatz", "--grants", "SymPy-1.1", "--due-date", "2021-01-01", '--notes', 'new note'],
      "ly_newprojectum has been added in projecta\n"),
-    (["helper", "u_milestone", "sb_firstprojectum", "--index", "4,5",
-      "--status", "converged", "--due-date", "2020-06-01", "--notes", "do this",
-      "do that", "--type", "meeting"],
-     "sb_firstprojectum has been updated in projecta\n"
-     ),
-    (["helper", "u_milestone", "sb"],
+    (["helper", "u_milestone", "--uuid_fragment", "sb_fir", "--status", "finished",
```

Comment:
Let's make the variable called `milestone_uuid` so it can be the full uuid, or a fragment of it.

Reply:
Will do. I like the idea of the user having the option to enter the full uuid, or the uuid fragment.